_i18n/en/resources/user-guides/join-monero-matrix.md

```diff
@@ -1,34 +1,35 @@
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
-Monero developers primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
+Most Monero developers primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
```

Comment:
```suggestion
Monero contributors primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
```

- "most" and "primarily" are redundant.
- Also may be good to switch "developers" to "contributors", as the socials are only for developers.